solutions/silver/cf-1879D.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: cf-1879D
+source: CF 1879D
+title: Sum of XOR Functions
+author: Justin Ji
+---
+
+<Warning title="Be careful with integer overflow!">
+
+Because we are working with very large numbers, the problem statement asks us to
+take the answer by some very large modulo. For more details on how to handle modular
+arithmetic, you can read the [Modular Arithmetic](https://usaco.guide/gold/modular)
+on this guide.
+
+</Warning>
```

Comment:
cool warning! this is usually taken for granted though, not sure if we should have this haha

Reply:
ok i'll remove it